cluster/log-dump/log-dump.sh

```diff
@@ -365,10 +365,12 @@ function dump_nodes_with_logexporter() {
   done
 
   failed_nodes=()
-  for node in "${NON_LOGEXPORTED_NODES[@]:-}"; do
-    echo "Logexporter didn't succeed on node ${node}. Queuing it for logdump through SSH."
-    failed_nodes+=("${node}")
-  done
+  if [[ -n "${NON_LOGEXPORTED_NODES}" ]]; then
+    for node in "${NON_LOGEXPORTED_NODES[@]}"; do
```

Comment:
Also here.

Reply:
Having it here means that we are ssh-ing to all nodes again after successful logexporter run.

I don't know why it is so - it works differently on my desktop.